api/features/views.py

```diff
@@ -205,6 +205,8 @@ def perform_destroy(self, instance):  # type: ignore[no-untyped-def]
         instance.delete()
 
     def get_serializer_context(self):  # type: ignore[no-untyped-def]
+        if getattr(self, "swagger_fake_view", False):
+            return None
```

Comment:
This should be done after the call to super and return the result of it. See [here](https://github.com/Flagsmith/flagsmith/pull/5724)